### ISSUE TITLE
[prefer-to-have-been-called-times] errors on machers other than `toHaveLength`

### ISSUE THREAD

**Issue by lukpsaxo** [jest-community/eslint-plugin-jest#1877]
With these new rules:
```
            'jest/prefer-to-have-been-called': 'error',
            'jest/prefer-to-have-been-called-times': 'error',
```

and the following assertion:

```
                expect(fn.mock.calls).toEqual([
                    ['x'],
                    ['y'],
                ]);
```

it gets auto fixed to:

```
                expect(fn).toHaveBeenCalledTimes(
                    [['x'], ['y']],
                );
```

**Comment by lukpsaxo** [comment]
The ideal solution would be to fix to:

```
expect(fn).toHaveBeenNthCalledWith(1, 'x');
expect(fn).toHaveBeenNthCalledWith(2, 'y');
```

The other solution would be to just not fix this - and arguably decide whether an error should be reported
